README.md

```diff
@@ -22,52 +22,35 @@ allprojects {
 }
 ```
 
-Then, add Mastodon4j dependencies to your module Gradle file:
+Then, add Bigbone dependencies to your module Gradle file:
 
 ```groovy
-implementation 'com.github.andregasser.mastodon4j:mastodon4j:$version'
-implementation 'com.github.andregasser.mastodon4j:mastodon4j-rx:$version'
+implementation 'com.github.andregasser.bigbone:bigbone:$version'
```

Comment:
Not entirely sure if this is allright.

Reply:
I guess this could be changed to point to 'social.bigbone:...' later, once we use mavenCentral. JitPack has a failing "custom domain" link on their front page, which probably isn't worth looking into